Sofascore/Sofascore/EventView.swift

```diff
@@ -0,0 +1,211 @@
+        statusLabel.font = ApplicationsFonts.micro
+        homeTeamScore.font = ApplicationsFonts.body
+        awayTeamScore.font = ApplicationsFonts.body
+        homeTeamName.font = ApplicationsFonts.body
+        awayTeamName.font = ApplicationsFonts.body
+        divider.backgroundColor = ApplicationColors.dividerColor
+    }
+
+    override func setupConstraints() {
+        eventHorizontalStack.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.bottom.equalToSuperview()
+            make.leading.equalToSuperview()
+            make.trailing.equalToSuperview()
+        }
+        
+        matchStatusVerticalStack.snp.makeConstraints { make in
+            make.top.equalTo(eventHorizontalStack.snp.top)
+            make.bottom.equalTo(eventHorizontalStack.snp.bottom)
+            make.leading.equalTo(eventHorizontalStack.snp.leading)
+            make.trailing.equalTo(divider.snp.leading)
+            make.width.equalTo(64)
+        }
+        
+        timeLabel.snp.makeConstraints { make in
+            make.top.equalTo(matchStatusVerticalStack.snp.top)
+        }
+        
+        statusLabel.snp.makeConstraints { make in
+            make.top.equalTo(timeLabel.snp.bottom)
+        }
+        
+        divider.snp.makeConstraints { make in
+            make.top.equalTo(eventHorizontalStack.snp.top).inset(8)
+            make.bottom.equalTo(eventHorizontalStack.snp.bottom).inset(8)
+            make.width.equalTo(1)
+        }
+        
+        teamInfoVerticalStack.snp.makeConstraints { make in
+            make.top.equalTo(eventHorizontalStack.snp.top)
+            make.bottom.equalTo(eventHorizontalStack.snp.bottom)
+            make.leading.equalTo(divider.snp.trailing).offset(16)
+            make.trailing.equalTo(scoreInfoVerticalStack.snp.leading)
+        }
+        
+        homeTeamLogo.snp.makeConstraints { make in
+            make.width.height.equalTo(16)
+        }
+        awayTeamLogo.snp.makeConstraints { make in
+            make.width.height.equalTo(16)
+        }
+        
+        homeTeamName.snp.makeConstraints { make in
+            make.height.equalTo(16)
+        }
+        
+        awayTeamName.snp.makeConstraints { make in
+            make.height.equalTo(16)
+        }
+        
+        scoreInfoVerticalStack.snp.makeConstraints { make in
+            make.top.equalTo(eventHorizontalStack.snp.top)
+            make.bottom.equalTo(eventHorizontalStack.snp.bottom)
+            make.leading.equalTo(teamInfoVerticalStack.snp.trailing)
+            make.trailing.equalTo(eventHorizontalStack.snp.trailing)
+        }
+        
+        homeTeamScore.snp.makeConstraints { make in
+            make.top.equalTo(scoreInfoVerticalStack.snp.top)
+            make.width.equalTo(32)
+        }
+        
+        awayTeamScore.snp.makeConstraints { make in
+            make.top.equalTo(homeTeamScore.snp.bottom)
+            make.width.equalTo(32)
+        }
```

Comment:
homeTeamScore i awayTeamScore su isti view-ovi 
Iako je vama zadatak da napravite 2 custom view-a: event i league, pozeljno je da ako primjetite repetitivan kod da to izdvojite npr u zaseban view. Mozes probati to ovdje